dev/update-lock-files.sh

```diff
@@ -9,5 +9,5 @@ TOP="${1:-$(dirname "$(dirname "$(readlink -f "$0")")")}"
 
 pushd "${TOP}" > /dev/null || exit 1
 poetry lock --no-update
-poetry export --dev --without-hashes --no-ansi --extras all > "${TOP}/requirements.txt"
+poetry export --with dev --without-hashes --no-ansi > "${TOP}/requirements.txt"
```

Comment:
This removes extras from requirements.txt. I think this is desirable, since there's a bunch of implicit system dependencies that show up if you include all the extras, whereas all the non-optional dependencies have wheels for every platform or are platform independent.